### PR TITLE
Color prompt and updated help

### PR DESCRIPTION
**Pull request by paulirish**
## help screen
With the addition of the new browsers, the help screen needed some love.  

Here's where I ended up: 
image

## color repl prompt
Also I brought back ansi escape codes to the prompt. 
I see back in https://github.com/Automattic/browser-repl/commit/d807b49c1d2ad10bfcd0aad162af19b4290e39be it was nuked, but this has since been fixed, as far as I can tell: https://github.com/joyent/node/issues/3860#issuecomment-54784252

But, I'm happy to revert this bit if you'd rather play it safe.

## action shot!
2x2 browser-repl grid of android4.1, ios 8, firefox 37, and android lollipop

image

### CODE REVIEW

File: repl.js [hunk 1]
```diff
@@ -132,7 +141,7 @@ function usage(){
 function start(){
   console.log('… ready!');
   var cmd = repl.start({
-    prompt: str + ' › ',
+    prompt: '\033[96m' + str + ' › \033[39m',
```
Comment (TooTallNate) [inline repl.js]: Maybe we could test if the version of node running supports the colored prompt or not? I believe that v0.12.2 was the first version to include the fix.

Comment (paulirish) [inline repl.js]: Good idea. Will do

**Comment by rauchg** [comment]
This is amazing 😍

**Comment by paulirish** [comment]
Cool. we should be sorted now. thx for the review and green-buttoning the other PRs!

## color prompt 
The bugfix for the ANSI escape characters (https://github.com/OhJeez/node/commit/2cbb876c3bb8317a1bc60460dea0fc1722c2206b) introduced a new method called `stripVTControlCharacters` for the readline module. 
So I'm just feature detecting that, and using the color if it passes. 
Progressive enhancement-style. :)

Comparison of prompt color in 0.12 and 0.10:
image

## connected browser details
Snuck this feature in. :)  
Once connected, we'll report which browser & version we have on the line. Especially nice when you're requesting the "latest" to verify that.
image

**Comment by TooTallNate** [comment]
Beautiful!

**Pull request merged**